### PR TITLE
Merge dev into main for release

### PR DESCRIPTION
**Pull request by superdingo101**

### CODE REVIEW

File: src/skylight-calendar-card.js [hunk 1]
```diff
@@ -1840,80 +1908,593 @@ class SkylightCalendarCard extends HTMLElement {
...
+            ...existingMetadata,
+            lastSuccessfulRefresh: existingMetadata.lastSuccessfulRefresh,
+            refreshFailed: true,
+            firstFailureAt: existingMetadata.firstFailureAt ?? Date.now(),
+            lastNetworkFailureRequest: generation
+          };
+        });
+        this.recomputeEventState();
         this.render();
+        return;
```
[the remark this answers is not in the document]
Comment (superdingo101) [reply src/skylight-calendar-card.js:2359]: @codex address that feedback.

Use the existing `_lastFetch` / `shouldRefreshEvents()` throttle when `_loadedEventRange` is null after a total calendar fetch failure. Do not mark a failed fetch range as successfully covered.

Prefer removing the unconditional `|| !this._loadedEventRange` fetch condition from `ensureEventsForCurrentRange()`, since `shouldRefreshEvents()` already triggers the initial fetch when `_lastFetch` is unset and permits retries after the existing 60-second interval.

Add a regression test proving that, after all calendars fail and `_loadedEventRange` remains null:

* immediate subsequent calls do not refetch,
* calls before the 60-second threshold do not refetch,
* a call after the threshold retries exactly once, and
* failed ranges are not represented as successful coverage.

Run `npm test` and `npm run check:build`, including the rebuilt committed bundle.
